src/storage/src/source/healthcheck.rs

```diff
@@ -235,8 +227,8 @@ impl Healthchecker {
                 .expect("status collection should not have errors");
             let row_vec = row.unpack();
             let row_source_id = row_vec[2].unwrap_str();
-            let row_worker_id = row_vec[5].unwrap_int64();
-            let row_status = row_vec[7].unwrap_str();
+            let row_worker_id = row_vec[3].unwrap_int64();
```

Comment:
I'm not a big fan of these raw numbers but I'm not sure they're worth fixing atm